src/packages/client/src/__tests__/integration/happy/blog-env-mssql/test.ts

```diff
@@ -0,0 +1,244 @@
+import sql from 'sql-template-tag'
+
+import { generateTestClient } from '../../../../utils/getTestClient'
+import { SetupParams, setupMSSQL } from '../../../../utils/setupMSSQL'
+
+describe('blog-env-mssql', () => {
+  let prisma: any = null // Generated Client instance
+  let requests: any[] = []
+
+  beforeAll(async () => {
+    const connectionString =
+      process.env.TEST_MSSQL_URI ||
+      'mssql://SA:Prisma1-prisma@localhost:1433/master'
+    const setupParams: SetupParams = {
+      connectionString,
+      dirname: __dirname,
+    }
+
+    await setupMSSQL(setupParams)
+
+    await generateTestClient()
+    const { PrismaClient } = require('./node_modules/@prisma/client')
+
+    prisma = new PrismaClient({
+      errorFormat: 'colorless',
+      __internal: {
+        measurePerformance: true,
+        hooks: {
+          beforeRequest: (r: any) => requests.push(r),
+        },
+      },
+      datasources: {
+        db: {
+          url: connectionString,
+        },
+      },
+      log: [
+        {
+          emit: 'event',
+          level: 'error',
+        },
+      ],
+    })
+  })
+
+  afterAll(async () => {
+    prisma!.$disconnect()
+  })
+
+  test('includes version in generated client', () => {
```

Comment:
Separated out tests into their own `test` blocks. I was referencing the postgres tests to figure out what to test.

IMO we should do this for them as well, since they're all in one `test` block right now. It might get hard to debug them in the future.